docs/en/users/06_Fever_API.md

```diff
@@ -0,0 +1,114 @@
+# FreshRSS - Fever API implementation
+
+## RSS clients
+
+There are many RSS clients existing supporting Fever APIs but they seem to understand the Fever API a bit differently. 
+If your favorite client doesn't work properly with this API, create an issue and we will have a look. 
+But we can ONLY do that for free clients, as we are am not willing to buy any RSS client. 
+
+### Usage & Authentication
+
+Before you can start to use this API you have to enable and setup API access, which is [documented here](https://freshrss.github.io/FreshRSS/en/users/06_Mobile_access.html).
+
+Please point your application to the URL of `fever.php` (e.g. `http://your-freshrss-url/api/fever.php`).
+
+Special client implementation:
+- The Press Android client needs (tested with 1.5.4) needs the additional file `fever-press.php` (use that file as endpoint in the Fever account setting)
+
+There is a drawback when using this plugin: the username and password combination that you have to use.
+You have to set the API password as **lowercased** md5 sum from the string 'username:password' (this limitation exists, because the Fever API was designed in a way which is incompatible with the authentication system used by FreshRSS).
+
+So if you use **kevin** as username and **freshrss** as password you have to set your FreshRSS API password to **4a6911fb47a87a77f4de285f4fac856d** (it MUST be lowercased, some tools seem to create it in uppercase which will not work!).
```

Comment:
What do the users have to type on the mobile app side? The plaintext password, right? If so, we can do some magic server-side to hide this complexity to the users.

Reply:
As explained below, we could save two API password but only showing one input in the user profile. Once like it is and once md5 hashed with username:password. Do you want to add that to the PR?

Reply:
I would like to find an approach that scales a bit better. For instance, we could have the username somewhere in the Fever API URL, such as `https://rss.example.net/api/fever.php/alice/`. Another solution could be to have a directory with text files `./data/fever/[md5].txt` and containing the name of respective accounts.

Reply:
If you come up with a solution that you prefer, please let me know. I can add it, but I don't want to change the core without some guidance. Privacy and security is a very important issue and I don't want to decide that by myself.

Reply:
Yes, I will do so. Let's check that the rest works fine first. I am slowed down by family activities for a couple of weeks

Reply:
Me too, sorry for not being more active, but free time is rare atm